src/o365/spo/commands/site/site-add.ts

```diff
@@ -466,27 +465,23 @@ class SpoSiteAddCommand extends SpoCommand {
     const options: CommandOption[] = [
       {
         option: '--type [type]',
-        description: 'Type of modern sites to list. Allowed values TeamSite|CommunicationSite|ClassicSite, default TeamSite',
+        description: 'Type of sites to list. Allowed values TeamSite|CommunicationSite|ClassicSite, default TeamSite',
```

Comment:
This should be `Type of site to add` 🙂

Reply:
Sometimes it looks that I am drunken during development

Reply:
Don't worry. Happens to all of us. That's why it helps to have another pair of eyes look at the code